### PR TITLE
Update onedrive-lti.md

### PR DESCRIPTION
**Pull request by msbemba**
Added another step after 9 stating that the remaining permission are needed. so another admin consent is required.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7686

File: microsoft-365/lti/onedrive-lti.md [hunk 1]
```diff
@@ -70,7 +70,8 @@ Integrating Microsoft OneDrive LTI with Canvas is a two step process. The first
 :::image type="content" source="media/OneDrive-LTI-20.png" alt-text="The LTI tenant registration page, which shows the JSON text and the text box the key should be copied into.":::
 
 9. Review and save your changes. A message will be displayed on successful registration.
-10. Your registration details can also be reviewed by selecting the **View LTI Tenants** button on the home page.
+10. Sign into the <a href="https://onedrivelti.microsoft.com/admin" target="_blank">Microsoft OneDrive LTI Registration Portal</a> and  Select the **Admin Consent** button and accept the remaining permissions.
```
Comment (JohanFreelancer9) [inline microsoft-365/lti/onedrive-lti.md]:
```suggestion
10. Sign into the <a href="https://onedrivelti.microsoft.com/admin" target="_blank">Microsoft OneDrive LTI Registration Portal</a>, select the **Admin Consent** button, and accept the remaining permissions.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MicrosoftHeidi Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@MicrosoftHeidi Gentle reminder.

**Comment by yogkumgit** [comment]
@MicrosoftHeidi Kindly merge this PR. Thanks

**Pull request merged**